sh/gitbak.sh

```
@@ -192,23 +233,21 @@ else
 if [ "$CONTINUE_SESSION" = "true" ]; then
     CREATE_BRANCH="false"
     echo "🔄 Continuing gitbak session on branch: $CURRENT_BRANCH"
 
-    # Escape special characters in the commit prefix for safer regex matching
     ESCAPED_PREFIX=$(echo "$COMMIT_PREFIX" | sed 's/[][\\/.*^$]/\\&/g')
 
-    # Set pipefail to catch errors in multi-command pipelines
-    # This makes the script fail if any command in a pipeline fails, not just the last one
+    # Set pipefail to catch errors in multi-command pipelines so that the script
+    # fails if any command in a pipeline is borked, not just the last one
     set -o pipefail 2>/dev/null || log "WARNING" "pipefail not supported in this shell"
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)